Telemetry/RulesTest.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net;
+using Helpers.Http;
+using Xunit;
+
+namespace Telemetry
+{
+    public class RulesTest
+    {
+        private readonly IHttpClient httpClient;
+        private const string TELEMETRY_ADDRESS = "http://127.0.0.1:9004/v1";
```

Comment:
nit: can this address be loaded from consolidated list in 'env.list' or from environment so that it is not hard coded?